pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import MDAnalysis as mda
+import pmda
+import dask
+from dask.delayed import delayed
+import time
+from joblib import cpu_count
+from pmda.parallel import ParallelAnalysisBase
+from pmda.util import timeit
+
+class RMSF(ParallelAnalysisBase):
+    """Parallel RMSF analysis.
+
+    Attributes
+    ----------
+    Parameters
+    ----------
+
+    Note
+    ----
+    At the moment, this class doesn't do anything.
+    """
+
+    def __init__(self, atomgroup):
+        u = atomgroup.universe
+        super(RMSF, self).__init__(u, (atomgroup,))
+        self._atomgroup = atomgroup
+        self._top = u.filename
+        self._traj = u.trajectory.filename
+
+    def _prepare(self):
+        self.sumsquares = np.zeros((self._atomgroup.n_atoms, 3))
+        self.mean = np.zeros((self._atomgroup.n_atoms, 3))
+
+    def _single_frame(self, ts, agroups, frame_index, sumsquares, mean):
+        """
+        Called for every frame.
+
+        Parameters
+        ----------
+        ts : int
+            current time step
+        agroups :
+            tuple of atomgroups that are updated to the current frame
```

Comment:
Document all other parameters. It's important for _you_ while your writing code as it explains to _you_ what everything should be. Don't write docs at the end, write them at the beginning as your own guidelines and to define what your code should do.